packages/dx-chart-core/src/plugins/controller-component/computeds.ts

```diff
@@ -0,0 +1,21 @@
+const difference = 0.9;
```

Comment:
We often name constants with the upper case and save it in `constants.ts` files.